src/pages/SignupHandler/index.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  ActionGroup,
+  Button,
+  Split,
+  SplitItem,
+  Title,
+} from '@patternfly/react-core';
+import { CheckIcon, CloseIcon } from '@patternfly/react-icons';
+import AuthLayout from '@/components/AuthLayout';
+import { routerRedux } from 'dva/router';
+import { connect } from 'dva';
+import styles from './index.less';
+
+@connect(user => ({
+  user: user.user,
+}))
+class SignupHandler extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      confPassword: '',
+      email: '',
+      variantVal: 'tertiary',
+      btnColor: 'black',
+    };
+  }
+
+  componentDidMount = () => {
+    this.disableSubmitBtn();
+  };
+
+  enableSubmitBtn = () => {
+    this.setState({
+      variantVal: 'primary',
+      btnColor: 'white',
+    });
+    const btn = document.getElementById('submitBtn');
```

Comment:
We should use React functionality here as opposed to the DOM API.

Reply:
Resolved